diff --git a/vendor/github.com/docker/engine-api/types/types.go b/vendor/github.com/docker/engine-api/types/types.go
index b6f9125..d3dc26e 100644
--- a/vendor/github.com/docker/engine-api/types/types.go
+++ b/vendor/github.com/docker/engine-api/types/types.go
@@ -260,6 +260,11 @@ type Info struct {
 	// running when the daemon is shutdown or upon daemon start if
 	// running containers are detected
 	LiveRestoreEnabled bool
+	Registries         []Registry
+}
+
+type Registry struct {
+	Name string
 }
 
 // PluginsInfo is a temp struct holding Plugins name
